woocommerce/woocommerce-gutenberg-products-block
# Fix: Mini Cart block: divide contents into three inner blocks

Fixes #4855 

This PR:
- Splits the filled mini cart contents into three inner blocks: Mini Cart Title, Mini Cart Products Table, and Mini Cart Footer.
- Renders filled Mini Cart contents block in the editor.
- Removes the `<Drawer>` title.
- Adds background support to the mini cart content block.
- Updates `mini-cart` template part file.
- Updates markup in the `MiniCartContents.php` to match with the component.
- Sets a maximum width for the Mini Cart Contents block in the Site Editor.
- Blocks interactive for Mini Cart Products Table and Footer blocks.




### Screenshots

<details>
<summary>Click to expand</summary>

#### Editor
![image](https://user-images.githubusercontent.com/5423135/146174085-3f7fa6cf-a682-43b4-8643-549c0e93ce00.png)

#### Front end
![image](https://user-images.githubusercontent.com/5423135/146174322-b7e53817-a1d8-48eb-97bc-1f07d0fcde3a.png)

</details>



### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Delete all customized mini-cart template parts in `/wp-admin/edit.php?post_type=wp_template_part`.
2. Delete the theme's mini-cart template part file.
3. Edit the Mini Cart template: Site Editor > Template Parts > Mini Cart.
4. See no error.
5. See the content of the filled mini cart renders in the editor.
6. Select the mini cart content block.
7. Set the background color.
8. See it works in the editor.
9. Save the change, see the background color works on the front end.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
